resources/docs/simulation/extra/migrating/anylogic/main-loop.md

```diff
@@ -17,6 +17,6 @@ In this simulation it’s the process of delivering oil from tankers, through th
 5. Which triggers fuel tankers to go to the storage agents load oil
 6. And deliver it to retailers
 
-![](https://lh6.googleusercontent.com/smDMCG9NqVz4Hg3VFdiQr95oNibC9mvxEYmxugc2ckHSkZ1vOd8qrUyodV3___N3UzSXpuVI9UpnYr6uOSmrN_DoOAvAOPfjl7-xYyIRUzEwHSg6GaBdQizKsLRWdj6i2F3NcSgy)
+![](https://s3.amazonaws.com/cdn-us1.hash.ai/site/docs/anylogic-oil-main-loop.png)
```

Comment:
@nshlapo For any URL that is `https://s3.amazonaws.com/cdn-us1.hash.ai/site/docs/anylogic-oil-main-loop.png` you can also just use `https://cdn-us1.hash.ai/site/docs/anylogic-oil-main-loop.png` (and should, as the former doesn't benefit from any content delivery acceleration).

Reply:
Fixed, thanks